crates/openapi/src/openapi_v2.rs

```diff
@@ -498,6 +498,7 @@ Never share your secret api keys. Keep them guarded and secure.
         api_models::payments::SamsungPayCardBrand,
         api_models::payments::SamsungPayTokenData,
         api_models::payments::PaymentsCancelRequest,
+        api_models::payments::PaymentsCancelPostCaptureRequest,
```

Comment:
Nit: You probably don't need it unless you're exposing the route in v2 OpenAPI spec...